RustvizParse/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "parse"
+version = "0.1.0"
+authors = ["Haochen Zhang <haochenz@Haochens-MacBook-Pro.local>"]
```

Comment:
add at least myself to the authors list (and any of the other students that have contributed to this code), and fix the email address, also the "edition" shouldn't be "2018"